pkg/services/accesscontrol/ossaccesscontrol/receivers.go

```diff
@@ -78,7 +78,7 @@ func (r ReceiverPermissionsService) SetDefaultPermissions(ctx context.Context, o
 	r.log.Debug("Setting default permissions for receiver", "receiver_uid", uid)
 	permissions := defaultPermissions()
 	clearCache := false
-	if user != nil && user.IsIdentityType(claims.TypeUser) {
```

Comment:
@kalleep should we add this in another PR?

@yuri-tceretian we will update so that service accounts, also get the `admin` privileges of created resources.